From aeba3ffaaa38de22c1af4907ce8d471d1320f7c2 Mon Sep 17 00:00:00 2001
From: Xiao Wang <jasowang@redhat.com>
Date: Thu, 10 Mar 2011 08:52:13 -0300
Subject: [PATCH 01/11] net: Add the missing option declaration of "vhostforce"

RH-Author: Xiao Wang <jasowang@redhat.com>
Message-id: <20110310085212.30250.47220.stgit@dhcp-91-7.nay.redhat.com.englab.nay.redhat.com>
Patchwork-id: 19848
O-Subject: [RHEL6.1 qemu-kvm PATCH] net: Add the missing option declaration of
	"vhostforce"
Bugzilla: 683295
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

Bugzilla: 683295
Test status: Test in my local desktop
Brew build: https://brewweb.devel.redhat.com/taskinfo?taskID=3166224

Add the missed "vhostforce" option.

Signed-off-by: Jason Wang <jasowang@redhat.com>
Acked-by: Michael S. Tsirkin <mst@redhat.com>
Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
(cherry picked from commit 96c94b298f99d6edf4e49d03cc8458f5b6e9d5f0)
---
 net.c |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 net.c |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/net.c b/net.c
index 3fc9c9f..2f498ae 100644
--- a/net.c
+++ b/net.c
@@ -1025,7 +1025,11 @@ static struct {
                 .name = "vhostfd",
                 .type = QEMU_OPT_STRING,
                 .help = "file descriptor of an already opened vhost net device",
-            },
+            }, {
+                .name = "vhostforce",
+                .type = QEMU_OPT_BOOL,
+                .help = "force vhost on for non-MSIX virtio guests",
+        },
 #endif /* _WIN32 */
             { /* end of list */ }
         },
-- 
1.7.4.1.230.gae447

